aip/0122.md

```diff
@@ -252,4 +252,7 @@ strictly necessary, use an `_id` suffix (e.g. `topic_id`).
 
 ## Changelog
 
-- **2019-07-30**: Changed the nested collection brevity suggestion from "may" to "should"
+- **2019-08-01**: Changed the examples from "shelves" to "publishers", to
+  present a better example of resource ownership.
```

Comment:
Possibly add that you've changed the example of peer resources from PubSub to books/shelves?

Reply:
Sure.